arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -455,6 +458,9 @@ public void resolveAtContextReEntry(Hub hub, CallFrame frame) {
           scenarioFragment.setScenario(CALL_PRC_FAILURE);
         }
         emptyCodeFirstCoupleOfAccountFragments(hub);
+        long gasAfterCall = frame.frame().getRemainingGas();
+        commonValues.gasNext(gasAfterCall);
+        hub.currentFrame().lastValidGasNext(gasAfterCall);
```

Comment:
doesn't at this point hub.currentFrame == frame ?

Reply:
I mean the frame of resolveAtContextReEntry(Hub hub, CallFrame frame)

Reply:
I'm not sure ...